caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php

```diff
@@ -1794,21 +1794,21 @@ protected function _get_ticket_price_row(
             'price_currency_symbol' => EE_Registry::instance()->CFG->currency->sign,
             'show_plus_or_minus'    => $default && empty($price)
                 ? ''
-                : ' style="display:none;"',
+                : 'display:none;',
             'show_plus'             => ($default && empty($price)) || ($price->is_discount() || $price->is_base_price())
-                ? ' style="display:none;"'
+                ? 'display:none;'
                 : '',
             'show_minus'            => ($default && empty($price)) || ! $price->is_discount()
-                ? ' style="display:none;"'
+                ? 'display:none;'
                 : '',
             'show_currency_symbol'  => ($default && empty($price)) || $price->is_percent()
-                ? ' style="display:none"'
+                ? 'display:none'
                 : '',
             'PRC_amount'            => $default && empty($price)
                 ? 0
                 : $price->get_pretty('PRC_amount', 'localized_float'),
             'show_percentage'       => ($default && empty($price)) || ! $price->is_percent()
-                ? ' style="display:none;"'
```

Comment:
this was a silly way to do this wasn't it?
should have been done via a css class, oh well 🤷🏻

Reply:
Yes, a lot better!